extensions/amp-accordion/1.0/test/test-amp-accordion.js

```diff
@@ -495,6 +495,37 @@ describes.realWin(
         expect(section2).to.not.have.attribute('expanded');
         expect(section3).to.have.attribute('expanded');
       });
+
+      it('should capture events in bento mode (w/o "on" attribute)', async () => {
+        const section1 = element.children[0];
+        const section3 = element.children[2];
+
+        // Set up section 1 to trigger expand of section 3 on expand
+        // and collapse of section 3 on collapse
+        const api = await element.getApi();
+        section1.addEventListener('expand', () => api.expand('section3'));
```

Comment:
Probably would be nice to just have a test where we check that the event has been delivered.

E.g.

```
const spy = sandbox.spy();
section1.addEventListener('expand', spy);
// do work
expect(spy).to.be.calledOnce.
```

Reply:
I added spies to this unit test.  Do you think it's too convoluted this way?

Reply:
Correct me if I'm wrong, but I interpreted the suggestion as to use spies instead of propagating events to an additional section. Or if doing both, separate them as distinct `it` blocks (one which checks spies, one which copies mimics events onto the other section).

Reply:
Sure, no problem.  I've updated the tests to two separate tests.